README.md

```diff
@@ -61,6 +61,7 @@ dt.compile('# My API\n...', function (error, compilationResult) {
 ### Arguments
 
 - (string) - API Description Document provided as string.
+- (string) - Original file name of the API Description Document. **Soon to be deprecated! See [#6][filename-deprecation]**
```

Comment:
"Soon to be deprecated", isn't this already deprecated, it's soon to be removed?